core/gui/src/app/workspace/service/workflow-graph/model/joint-graph-wrapper.ts

```diff
@@ -841,18 +712,6 @@ export class JointGraphWrapper {
     return cell.attributes.z || 0;
   }
 
-  /**
-   * This method sets the cell's layer (z attribute) to the given layer.
-   * A cell can be an operator, a link, or a group element.
-   */
-  public setCellLayer(cellID: string, layer: number): void {
-    const cell: joint.dia.Cell | undefined = this.jointGraph.getCell(cellID);
-    if (!cell) {
-      throw new Error(`cell with ID ${cellID} doesn't exist`);
-    }
-    cell.set("z", layer);
-  }
```

Comment:
we still need operator and link layer?

Reply:
After removing the operator group, this function is not being used anywhere, and I believe we only have a single layer.